idseq/cli.py

```diff
@@ -204,5 +182,58 @@ def network_err_text():
         "needs to be re-uploaded (under a different name).")
 
 
+def _bulk_upload(args, headers):
```

Comment:
If you want to make it "private" use double underscore.

Reply:
I believe the convention Python is to use a single underscore unless name mangling is needed. See https://stackoverflow.com/questions/6930144/underscore-vs-double-underscore-with-variables-and-methods . That being said, I recognize that the convention in this repo is to not use any underscores, so I'd be ready to remove it.